config/jail.conf

```diff
@@ -256,6 +256,9 @@ logpath  = %(auditd_log)s
 
 [apache-auth]
 
+# To also protect web apps with default message patterns,
+# use the following filter :
+#filter  = apache-auth[extendregex="<protect_web_apps>"]
```

Comment:
I am still wondering @sebres how big of an impact would be just to have those regexes enabled? 
Are we spending most of the time regex matching or doing other logistics?
Making it optional as in here makes those regexes not tested anyhow so potentially making them not operating correctly or even harmful.